src/libs/actions/Policy/Policy.ts

```diff
@@ -4312,6 +4312,37 @@ function getAdminPoliciesConnectedToNetSuite(): Policy[] {
     return Object.values(allPolicies ?? {}).filter<Policy>((policy): policy is Policy => !!policy && policy.role === CONST.POLICY.ROLE.ADMIN && !!policy?.connections?.netsuite);
 }
 
+function getApproverPolicies(): Policy[] {
+    return Object.values(allPolicies ?? {}).filter<Policy>((policy): policy is Policy => {
+        if (!policy || !sessionEmail || policy.type === CONST.POLICY.TYPE.PERSONAL) {
+            return false;
+        }
+
+        const isPolicyApprover = policy.approver === sessionEmail;
+        const isSubmittedTo = Object.values(policy.employeeList ?? {}).some((employee) => {
+            return employee.submitsTo === sessionEmail;
+        });
+
+        return isPolicyApprover || isSubmittedTo;
+    });
+}
+
+function getExporterPolicies(): Policy[] {
+    return Object.values(allPolicies ?? {}).filter<Policy>((policy): policy is Policy => {
+        if (!policy || !sessionEmail) {
+            return false;
+        }
+
+        const isIntacctExporter = policy.connections?.intacct?.config?.export.exporter === sessionEmail;
+        const isNetSuiteExporter = policy.connections?.netsuite?.options.config?.exporter === sessionEmail;
+        const isQuickbooksDesktopExporter = policy.connections?.quickbooksDesktop?.config?.export.exporter === sessionEmail;
+        const isQuickbooksOnlineExporter = policy.connections?.quickbooksOnline?.config?.export.exporter === sessionEmail;
+        const isXeroExporter = policy.connections?.xero?.config?.export.exporter === sessionEmail;
+
+        return isIntacctExporter || isNetSuiteExporter || isQuickbooksDesktopExporter || isQuickbooksOnlineExporter || isXeroExporter;
+    });
+}
```

Comment:
Same as my comment on `getAllSharedPolicies`. Let's remove these methods and pass the policy list as a param and then filter on it